dist/esm/package.json

```diff
@@ -1,3 +0,0 @@
-{
-  "type": "module"
-}
```

Comment:
please leave this file as is, if you really want to remove it, make a separate MR with this...

Reply:
Yeah, I think i just deleted the whole dist library because i thought it was auto generated :). Will add it back in.